packages/buidler-core/src/internal/buidler-evm/provider/provider.ts

```diff
@@ -69,15 +74,15 @@ export class BuidlerEVMProvider extends EventEmitter
     super();
   }
 
-  public async send(method: string, params: any[] = []): Promise<any> {
+  public async request(args: RequestArguments): Promise<unknown> {
     const release = await this._mutex.acquire();
 
     try {
-      if (this._loggingEnabled && !PRIVATE_RPC_METHODS.has(method)) {
-        return await this._sendWithLogging(method, params);
+      if (this._loggingEnabled && !PRIVATE_RPC_METHODS.has(args.method)) {
+        return await this._sendWithLogging(args.method, args.params as any[]);
```

Comment:
How can we be sure that `args.params` is an array?

Reply:
Yeah, I think we should throw if an object is received.